core/src/main/scala/kafka/server/FetchSession.scala

```diff
@@ -526,11 +532,11 @@ class IncrementalFetchContext(private val time: Time,
           }
         }
         if (hasInconsistentTopicIds) {
-          FetchResponse.of(Errors.INCONSISTENT_TOPIC_ID, 0, session.id, new FetchSession.RESP_MAP, Collections.emptyMap())
+          FetchResponse.of(Errors.INCONSISTENT_TOPIC_ID, 0, session.id, new FetchSession.RESP_MAP)
```

Comment:
Since we moved an inconsistent topic ID check to the FetchResponse.of method we may want to remove this and similar checks in other contexts